docs/usage/sc_AIRRseq/single_cell_tutorial.md

```diff
@@ -0,0 +1,156 @@
+
+
+## Fastq format
+### Datasets
+For this tutorial we will use subsampled blood single-cell TCR sequencing data of one subject generated from the 10x Genomic platform. The links to the fastq files are in the samplesheet. 
+
+### Preparing samplesheet, gene reference and configuration file
+To run the airrflow pipeline on single cell TCR or BCR sequencing data from fastq files, we need to prepare samplesheet, gene reference and configuration file in advance. Details on the requied columns of a samplesheet are available [here](https://nf-co.re/airrflow/usage#input-samplesheet). 
+
+The prepared samplesheet for this tutorial is [here](sample_data_code/10x_sc_raw.tsv) and a prepared configuration file is [here](sample_data_code/resource.config). Download these two files to the directory where you intend to run the airrflow pipeline.
+
+> [Tip] 
+> Before setting memory and cpus in the configuration file, we recommend verifying the available memory and cpus on your system. Otherwise, exceeding the system's capacity may result in unexpected errors. 
+
+Gene reference can be accessed at the [10x Genomics website](https://www.10xgenomics.com/support/software/cell-ranger/downloads). Both human and mouse V(D)J references are available. Download the reference that corresponds to the species of your dataset. 
+
+
+
+### Running airrflow
+With all the files ready, it's time to run the airrflow pipeline. 
+
+```bash
+nextflow run nf-core/airrflow -r 4.2.0 \
+-profile docker \
+--mode fastq \
+--input 10x_sc_raw.tsv \
+--library_generation_method sc_10x_genomics \
+--reference_10x refdata-cellranger-vdj-GRCh38-alts-ensembl-7.1.0 \
+-c resource.config \
+--clonal_threshold 0 \    
+--outdir sc_from_fastq_results \
+-resume
+```
+
+Of course you can wrap all your code in a bash file. We prepared one for you and it's available [here](sample_data_code/airrflow_sc_from_fastq.sh).
+With the bash file, it's easy to run the pipeline with a single-line command. 
+
+```bash
+bash airrflow_sc_from_fastq.sh
+```
+
+By default, clonal_threshold is set to be 'auto', allowing the Hamming distance threshold of junction regions to be determined automatically. For BCR data, we recommend using this default setting. After running the pipeline, review the automatically calculated threshold to make sure it is appropriate. If the threshold is unsatisfactory, you can re-run the pipeline with a manually specified clonal_threshold. 
+In this tutorial, since the samples are TCRs, which do not have somatic hypermutation, clones are defined strictly by identical CDR3s. For this reason, we set the clone-threshold parameter to 0. 
+
+
+## Understanding the results
+
+After running the pipeline, several reports are generated under the result folder. 
+
+![example of result folder](tutorial_images/airrflow_result_folder_example.png)
```

Comment:
Just to make it easier to maintain, in case the folder names or structure changes, I would suggest to list the directories and the most important subdirectories with text like that:

```
Airrflow_report.html
- clonal_analysis
- repertoire_analysis
- qc-filtering
etc.
```